storj/model.py

```diff
@@ -250,6 +253,77 @@ def address(self):
         return self.keypair.address(use_uncompressed=False)
 
 
+class IdecdsaCipher(Object):
+    """
+    Tools for en-/decrypt private key to/from id_ecdsa
+    """
+
+    def __init__(self):
+        pass
+
+    def pad(self, data):
+        padding = 16 - len(data) % 16
```

Comment:
missing docstring.
you don't use `self` on the function body so you could move this function outside of the class.